doc/experiment.rst

```diff
@@ -12,316 +38,231 @@ Jupyter notebook, interactive python session, or Python file and use the
+to the ``Experiment`` initialization.
 
-.. code-block:: Python
-
-  from smartsim import Experiment
-
-  exp = Experiment("hello_world_exp", launcher="slurm")
-  srun = exp.create_run_settings(exe="echo", exe_args="Hello World!")
-
-  # helper methods for configuring run settings are available in
-  # each of the implementations of RunSettings
-  srun.set_nodes(1)
-  srun.set_tasks(32)
-
-  model = exp.create_model("hello_world", srun)
-  exp.start(model, block=True, summary=True)
-
-  print(exp.get_status(model))
-
-The above will run ``srun -n 32 -N 1 echo Hello World!``, monitor its
-execution, and inform the user when it is completed. This driver script can be
-executed in an interactive allocation, or placed into a batch script as follows:
+.. code-block:: python
 
-.. code-block:: bash
+    from smartsim import Experiment
+
+    exp = Experiment("name-of-experiment", launcher="local")  # local launcher
+    exp = Experiment("name-of-experiment", launcher="slurm")  # Slurm launcher
+    exp = Experiment("name-of-experiment", launcher="pbs")    # PBSpro launcher
+    exp = Experiment("name-of-experiment", launcher="cobalt") # Cobalt launcher
+    exp = Experiment("name-of-experiment", launcher="lsf")    # LSF launcher
+    exp = Experiment("name-of-experiment", launcher="auto")   # auto-detect launcher
+
+
+=========
+ Entities
+=========
+
+Defining workflow stages requires the utilization of functions
+associated with the ``Experiment`` object. The ``Experiment`` object
+is intended to be instantiated once and utilized throughout
+the workflow runtime. In the following content, we define five
+entities created by an experiment: ``Orchestrator``, ``Model``, ``Ensemble``,
+``RunSettings``, and ``BatchSettings``, along with a discussion of their relationships.
+
+Entity Relationship Diagram
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Below is an Entity Relationship Diagram using Crow's Foot Notation.
+Moving from the top to the bottom of the diagram, you will first notice that
+an Experiment can have none or many ``Orchestrators``, ``Models``, and ``Ensembles``.
+The Experiment class is responsible for all entity initialization functions:
+
+1. ``Experiment.create_orchestrator()`` :raw-html:`&rarr; ``Orchestrator``
+2. ``Experiment.create_model()`` :raw-html:`&rarr; ``Model``
+3. ``Experiment.create_ensemble()`` :raw-html:`&rarr; ``Ensemble``
+4. ``Experiment.create_run_settings()`` :raw-html:`&rarr; ``RunSettings``
+5. ``Experiment.create_batch_settings()`` :raw-html:`&rarr; ``BatchSettings``
+
+.. |SmartSim ERD| image:: images/edr.png
+  :width: 700
+  :alt: Alternative text
 
-    #!/bin/bash
-    #SBATCH --exclusive
-    #SBATCH --nodes=1
-    #SBATCH --ntasks-per-node=32
-    #SBATCH --time=00:10:00
+|SmartSim ERD|
+
+Moving on to the ``Model`` entity class, you will notice that a ``Model`` requires a
+minimum and maximum of one ``RunSetting`` object. However, it accepts a minimum of zero
```

Comment:
I think "a minimum and maximum of one" can be replaced by "exactly one", while "a minimum of zero and a maximum of one" can be replaced by "at most one", to keep the tone more colloquial.